### ISSUE TITLE
Mass-Replace AI Explosives

### ISSUE THREAD

**Issue by Joe-Banks** [Crowdedlight/Crows-Zeus-Additions#53]
A module that would, when activated, replace all of the Grenades + Secondary rounds in an AI unit's inventory with the base-game White Smoke. Useful for mitigating issues with using base-game and modded AI compositions for enemy/neutral forces during Zeus ops, that can be caused by modders not intending certain factions to be used as enemies (and thus filling their inventory with red smoke or a million frag grenades). To avoid abuse, this should not function on players.

**Comment by Crowdedlight** [comment]
three categories. Grenades, explosives, launcher checkboxes. 

Should set a public var on each AI unit, so if a unit has been "stripped" and its run again with same settings, then it is skipped instead for performance. As we are gonna iterate all AI units. If different settings it should be processed.

Check how we did the pause between units in body-cleanup script and if we spawned it. As ideally a few ms between each unit process would be better as its a longer process and we don't want to freeze the zeus while its running. 

Should do global hint when done for the zeus that started it.
